src/jobs/build_push_image.yml

```diff
@@ -73,6 +73,15 @@ parameters:
   image_suffix:
     type: string
     default: ""
+  repository_version:
+    description: 'The version of the ricardo-orbs from which to get the Dockerfiles, it can be a tag or a branch.'
+    type: string
+    default: 'v7.6.0'
```

Comment:
shouldn't the default be also `main` like the default is in the docker fetch dockerfile command?